bigtable/client/table.h

```diff
@@ -224,6 +230,24 @@ class Table {
                          std::vector<Mutation> true_mutations,
                          std::vector<Mutation> false_mutations);
 
+  /**
+   * Return a sample of the row keys in the table, including approximate data
+   * sizes.
+   *
+   * @return a collection of SampleRowKeys. (The type of collection is specified
```

Comment:
Remove the parenthesis.  You can just say:

```C++
/**
  * Sample of the row keys in the table, including approximate data sizes.
  *
  * The application can specify the collection type, for example:
  * @code
  * auto as_vector = table.SampleRows<std::vector>();
  * auto as_list = table.SampleRows<std::list>();
  * @endcode
  */
```

Reply:
Done.